app/src/main/java/org/liberty/android/fantastischmemo/ui/DatabaseMerger.java

```diff
@@ -121,18 +122,10 @@ public void doHeavyTask() throws Exception {
                 }
 
                 public void doUITask(){
-                    new AlertDialog.Builder(DatabaseMerger.this)
-                        .setTitle(R.string.merge_success_title)
-                        .setMessage(R.string.merge_success_message)
-                        .setPositiveButton(R.string.back_menu_text, new DialogInterface.OnClickListener(){
-                            public void onClick(DialogInterface arg0, int arg1){
-                                Intent resultIntent = new Intent();
-                                setResult(Activity.RESULT_OK, resultIntent);
```

Comment:
I think we want to set result OK in result intent to indicate the parent we want to refresh.

Reply:
Done.